packages/framework/presence/src/systemWorkspace.ts

```diff
@@ -74,14 +82,17 @@ class SystemWorkspaceImpl implements PresenceStatesInternal, SystemWorkspace {
 	public constructor(
 		clientSessionId: ClientSessionId,
 		private readonly datastore: SystemWorkspaceDatastore,
-		public readonly events: IEmitter<Pick<PresenceEvents, "attendeeJoined">>,
+		public readonly events: IEmitter<
+			Pick<PresenceEvents, "attendeeJoined" | "attendeeDisconnected">
+		>,
 	) {
 		this.selfAttendee = {
 			sessionId: clientSessionId,
 			order: 0,
-			currentConnectionId: () => {
+			connectionId: () => {
 				throw new Error("Client has never been connected");
 			},
+			status: () => SessionClientStatus.Disconnected,
```

Comment:
This is an example of where `status` being a function seems odd. It seems odd to set it to a function that returns a static value, then immediately assign it a new function (below) that returns a different static value.

If you want to leave it a function I would call it `getStatus()`.

Reply:
I agree with this being a bit odd, I notice we do something similar with currentconnectionId so I just followed the existing pattern. I think renaming here to `getStatus()` makes sense.

Reply:
> We used function for connectionId and should go the same for both. So -> function.

I just noticed Jason's comment from an earlier review. I think a getter makes for a more ergonmic dev surface for API consumers. That said, I agree with Jason's reasoning and see no reason to introduce inconsistency. I still think `.getStatus` is better, because the verb helps clarify it's an operation, and make it less likely to type `.status === 'disconnected'` when you really meant `.status()`. TypeScript will help prevent that sort of mistake, though. I say go with getStatus() and if Jason prefers .status then we can change it then.

Reply:
`.status()` has to be the name or `.connectionId()` needs to become `.getConnectionId()` for symmetry.
I don't think we require `get` prefix on everything; so, just `status()`. If that doesn't feel right, then API council can help.
My requirements: (1) must be method and (2) consistency across APIs.